src/views/RunPage.vue

```diff
@@ -1,16 +1,17 @@
 <template>
-  <div>Running</div>
+    <div>Running</div>
 </template>
 
 <script setup lang="ts">
-import { onMounted, onUnmounted } from 'vue'
+import { onMounted, onUnmounted } from 'vue';
+
 onMounted(() => {
-  console.log('start Capturing')
-})
+    console.log('start Capturing');
+});
 
 onUnmounted(() => {
-  console.log('stop Capturing - close all')
-})
+    console.log('stop Capturing - close all');
+});
 </script>
 
-<style lang="scss"></style>
+<style lang="scss" scoped></style>
```

Comment:
Is this view required at all? "Run" should just be a toggleable button imo.

Reply:
I thought it could be nice because the in the sidebar where normally the sources are there is a even clearer feedback what is happening and if something goes wrong what is going wrong.

Reply:
Not quite sure about this. We'll discuss in the next meeting.

Reply:
So can this be resolved or should we wait ?

Reply:
Resolving for now, #31 addresses it.